packages/system-message-react/package.json

```
@@ -32,13 +32,13 @@
         "build:types": "tsc -p tsconfig-for-declarations.json",
         "build:scripts": "node ./esbuild.prod.mjs",
         "build": "run-s build:*",
-        "test": "jest --testMatch '**/alert-message-react/**/*.test.+(ts|tsx|js)' --config=../../jest.config.js",
+        "test": "jest --testMatch '**/system-message-react/**/*.test.+(ts|tsx|js)' --config=../../jest.config.js",
         "dev:js": "ESBUILD_WATCH=true node ./esbuild.dev.mjs",
         "dev:server": "nodemon ./dist/server.js",
         "dev": "run-p dev:*"
     },
     "dependencies": {
-        "@fremtind/jkl-alert-message": "^9.0.12",
+        "@fremtind/jkl-system-message": "^9.0.12",
```

```suggestion
        "@fremtind/jkl-system-message": "^1.0.0-alpha.0",
```
Siden vi endrer versjonsnummer på stil pakken også. Denne vil også automatisk bli oppdatert.